src/libfsm/minimise.c

```diff
@@ -368,10 +382,255 @@ dump_ecs(FILE *f, const struct min_env *env)
 #endif
 }
 
-static void
-populate_initial_ecs(struct min_env *env, const struct fsm *fsm)
+#define PARTITION_BY_END_STATE_DISTANCE 1
```

Comment:
What's the plan to eventually resolve this?

Reply:
We could just remove it later. I wanted to leave it easy to enable/disable for benchmarking, particularly as we try to formalize the performance impact.